index.html

```diff
@@ -219,27 +265,268 @@ <h2>Badge</h2>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="card-code-container">
+                        <iframe src="https://www.thiscodeworks.com/embed/620760b63d90130015398beb"
+                            style="width: 100%; height: 489px;" frameborder="0"></iframe>
+                    </div>
+
+                    <h3>Card with text Overlay</h3>
+                    <div class="card-container">
+                        <div class="card">
+                            <div class="container">
+                                <div class="image img-text">
+                                    <img src="src/images/card/image.png" alt="the-alchemist-book-cover" class="img" />
+                                    <div class="text text-overlay">
+                                        <h2>The Alchemist</h2>
+                                    </div>
+                                </div>
+                                <div class="para">
+                                    The Alchemist is a novel by Brazilian author Paulo Coelho which was first published
+                                    in
+                                    1988.
+                                </div>
+                                <div class="card-buttons">
+                                    <div class="card-btn">
+                                        <button type="submit" class="btn1 card-btn-items">
+                                            Read
+                                        </button>
+                                        <button type="submit" class="btn1 card-btn-items">
+                                            Bookmark
+                                        </button>
+                                    </div>
+                                    <div class="card-icons">
+                                        <i class="fa fa-heart-o"></i>
+                                        <i class="fa fa-share-alt"></i>
+                                        <i class="fa fa-ellipsis-v"></i>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="card-code-container">
+                        <iframe src="https://www.thiscodeworks.com/embed/6207615a3d90130015398bed"
+                            style="width: 100%; height: 463px;" frameborder="0"></iframe>
+                    </div>
+
+
+                    <h3>Card with Dismiss</h3>
+                    <div class="card-container">
+                        <div class="card">
+                            <div class="container">
+                                <div class="image">
+                                    <img src="src/images/card/image.png" alt="the-alchemist-book-cover" class="img" />
+                                    <div class="img-badge1">X</div>
```

Comment:
Class names need to be proper and self-explanatory, this will help you and others to understand code quickly in the future and will also make debugging fast.
Eg: Instead of "img-badge1", it can be named as "dismiss-badge".

Reply:
okay. I will keep that in mind